### PR TITLE
Adding the i18n keys for the voucher status

### PR DESCRIPTION
**Pull request by JohnTitor2036**
### Screenshots

English
image

French
image

### CODE REVIEW

**Comment by JohnTitor2036** [comment]
Needed for the [pull request #175 of the frontend project](https://github.com/ourjapanlife/findadoc-frontend/pull/175) :)

File: locales/en.json [hunk 1]
```diff
@@ -63,6 +63,10 @@
             "submitReport": "Report",
             "title": "Report this data"
         },
+        "voucher": {
+            "required" : "Required",
```
Comment (theyokohamalife) [inline locales/en.json:67]: Can you remove the space before the colon? Example: `"required":  "Required",`

File: locales/en.json [hunk 2]
```diff
@@ -63,6 +63,10 @@
...
+            "notRequired" : "Not Required"
```
Comment (theyokohamalife) [inline locales/en.json:68]: Please remove the space here too.

**Review by theyokohamalife — CHANGES_REQUESTED** [review]
Thank you for doing this so quickly! Just a couple small changes requested.

**Comment by JohnTitor2036** [comment]
I've removed the extra space, thank you!

**Review by ann-kilzer — COMMENTED** [review]
LGTM 🌟

**Comment by ann-kilzer** [comment]
Let's make sure to get a review before merging in the future :)

**Comment by JohnTitor2036** [comment]
@ann-kilzer My mistake, I'll do that next time.

**Pull request merged**